cfgov/preprocessed/css/molecules/call-to-action.less

```diff
@@ -0,0 +1,41 @@
+/* topdoc
+  name: Call to Action
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+		<div class="m-call-to-action">
+		    <p class="short-desc">
+		    </p>
+	        <a class="btn btn__full m-call-to-action_btn" href="/">
+	        </a>
+		</div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .m-call-to-action
+            .short-desc
+            .btn btn__full.m-call-to-action_btn
+  tags:
+    - cfgov-molecules
+*/
+
+.m-call-to-action {
+
+    .respond-to-range(@tablet-min, @tablet-max, {
+        .grid_column(6);
+    });
+
+    .btn {
```

Comment:
Nope, not needed. Removed in https://github.com/cfpb/cfgov-refresh/commit/099a9d21cddc35d98430aa93e93ce1fb6c6af6d0.